Library/Formula/ebook-tools.rb

```diff
@@ -1,6 +1,6 @@
 class EbookTools < Formula
   desc "Access and convert several ebook formats"
-  homepage "http://sourceforge.net/projects/ebook-tools"
+  homepage "https://ebook-tools.sourceforge.net/"
```

Comment:
Again, ends up on https://sourceforge.net/projects/ebook-tools/ if you follow the directs.